src/content/themes/starlog.md

```diff
@@ -0,0 +1,34 @@
+---
+title: "Starlog"
+description: "A light-weight theme for your release notes"
+image: "/src/content/themes/_images/starlog-hero.webp"
+images:
+  - "/src/content/themes/_images/starlog-1.webp"
+  - "/src/content/themes/_images/starlog-2.webp"
+author:
+  url: "https://github.com/withastro"
+  name: "withastro"
+  avatar: "https://avatars.githubusercontent.com/u/44914786?v=4"
+categories:
+  - "official"
+  - "docs"
+  - "recent"
+repoUrl: "https://github.com/withastro/astro/tree/main/examples/starlog"
+tools:
+  - "sass"
+publishDate: "Jan 2, 2024"
```

Comment:
just a note that I updated this timestamp!  I went ahead and added the "recent" category and "new" badge while I was here too, our weekly script will add that for any theme published within 30 days

Reply:
Thanks!